.github/workflows/spellcheck.yml

```diff
@@ -277,6 +290,8 @@ jobs:
 #W:HTTPS
 #W:https
 #W:Huawei
+#W:IANA
+#W:ICONV
 #W:IDN
 #W:IDNA
 #W:IETF
```

Comment:
"ipod", if it's the music device, should be "iPod". I actually can't find some of these words in the repo so I can't see the context.